gazebo/rendering/Camera.cc

```diff
@@ -1094,6 +1116,30 @@ double Camera::FarClip() const
     return 0;
 }
 
+//////////////////////////////////////////////////
+double Camera::ImageFocalLengthX() const
+{
+  return this->dataPtr->cameraIntrinsicMatrix(0, 0);
+}
+
+//////////////////////////////////////////////////
+double Camera::ImageFocalLengthY() const
+{
+  return this->dataPtr->cameraIntrinsicMatrix(1, 1);
+}
+
+//////////////////////////////////////////////////
+double Camera::ImageOpticalCentreX() const
+{
+  return this->dataPtr->cameraIntrinsicMatrix(0, 2);
+}
+
+//////////////////////////////////////////////////
+double Camera::ImageOpticalCentreY() const
+{
+  return this->dataPtr->cameraIntrinsicMatrix(1, 3);
```

Comment:
I _think_ the element we want is `(1, 2)`, no?

Reply:
yeah right, makes me wonder how unit tests were working. I will take a look. Thanks for pointing it out.